frontend.py

```diff
@@ -109,7 +106,7 @@ def create_robot_sprite(robot):
     return robot_sprite
 
 
-def draw_state(state, winner_time, window):
+def draw_state(state, winner_time, available_robots, window):
```

Comment:
Priznam se, ze mi neni jasne, proc se nove parametry funkci davaji doprostred, misto aby nakonec, ale jelikoz na funkcnost to vliv nema, nevidim v tom blocker.

Reply:
člověče nevím proč :-D Je nějaká konvence že se to má vždy dávat na konec? :)

Reply:
Rekla bych, ze spis na cit: cim min zmen, tim lip. Kdyz je funkce "nakresli_stav", pak mi nejak sedi, ze prvni parametr je onen "stav", a druhy "okno", kam se vykresli. Ale dokud nezavadis pojmenovane argumenty, je to asi wsio rawno, to je takova poznamka bokem: proste by me to nenapadlo udelat tak :)

Reply:
Takto když to říkáš, dává to větší smysl, já zase osobně nějak to window cpala prostě nakonec citově nevím proč ale máš pravdu :-)